src/components/common/PriceCalculation.vue

```diff
@@ -101,6 +110,26 @@ export default {
       }
       return null;
     },
+
+    discount() {
+      let relativeDiscount = null;
+      let absoluteDiscount = null;
+      if (this.cartLike.discountCodes) {
+        this.cartLike.discountCodes.forEach((e) => {
+          e.discountCode.cartDiscounts.forEach((code) => {
+            // eslint-disable-next-line
+            if (code.value.__typename === 'RelativeDiscountValue') {
+              relativeDiscount = code.value.permyriad * 0.01;
+              // eslint-disable-next-line
+            } else if (code.value.__typename === 'AbsoluteDiscountValue') {
+              // eslint-disable-next-line
```

Comment:
What's the reason to disable all these previous lines? eslint should only be disabled in exceptional cases, such as with `i18n` max-len (because there's no way around and it's not code in the end), and always disabling only for the specific problem, not everything.

Reply:
it was saying that I shouldn't use underlines but there isn't another way as far as I know

Reply:
But even if there's no way around, you never should disable the whole eslint: https://stackoverflow.com/questions/44126983/eslint-unexpected-dangling-in-place-no-underscore-dangle